hubtools/src/bootleby.rs

```diff
@@ -89,8 +90,11 @@ pub fn bootleby_to_archive(
     board: String,
     name: String,
     gitc: String,
+    epoc: Option<u32>,
```

Comment:
API-wise, it would be cleaner to pass a `u32` here, so the caller has to decide how to handle the case when it's missing.